### PR TITLE
Add SIP Outbound video Flag and PlayDTMF function with API endpoints

### PR DESCRIPTION
**Pull request by geekchick**
Hi!

These changes are for the following JIRA tickets:

Add SIP Video Outbound flag to Python sDK: https://jira.vonage.com/browse/DEVX-5646
Add "Play DTMF" API to Python SDK: https://jira.vonage.com/browse/DEVX-5652

### CODE REVIEW

**Review by calthoff — COMMENTED** [review]
Hey Tonya! Looks good! It is not clear to me (someone still familiarizing myself with this SDK) exactly what a "force mute action" is, though. Maybe consider adding a sentence clarifying what that means in the docstring? Also, it looks like you are defining the variable observeForceMute but not using it for anything unless I am missing something.

**Comment by geekchick** [comment]
> Hey Tonya! Looks good! It is not clear to me (someone still familiarizing myself with this SDK) exactly what a "force mute action" is, though. Maybe consider adding a sentence clarifying what that means in the docstring?

Hi Cory! Ok, sounds good! Force Mute allows a moderator to force clients to mute their audio in streams they publish. So I can add something like that to elaborate on it.

File: opentok/opentok.py [hunk 1]
```diff
@@ -1447,3 +1460,41 @@ def mute(self, session_id: str, stream_id: str= "", options: dict = {}) -> reque
...
 
+    def play_dtmf(self, session_id: str, connection_id: str, digits: str, options: dict = {}) -> requests.Response:
+        """
+        Plays a DTMF string into a session or to a specific connection
+
+        :param session_id The ID of the OpenTok session that the participant being called
+        will join
+
+        :param connection_id An optional parameter used to send the DTMF tones to a specific
+        connectoiin in a session.
```
Comment (calthoff) [inline opentok/opentok.py:1471]: Typo: connection

File: opentok/opentok.py [hunk 2]
```diff
@@ -1042,6 +1050,7 @@ def dial(self, session_id, token, sip_uri, options=[]):
         """
         payload = {"sessionId": session_id, "token": token, "sip": {"uri": sip_uri}}
         observeForceMute = False
```
Comment (calthoff) [inline opentok/opentok.py:1052]: Why define these variables? I don't see you use them later.

File: opentok/opentok.py [hunk 3]
```diff
@@ -1447,3 +1460,41 @@ def mute(self, session_id: str, stream_id: str= "", options: dict = {}) -> reque
                 ("There was an error thrown by the OpenTok SDK, please check that your session_id {0} and stream_id (if exists) {1} are valid").format(
                     session_id, stream_id))
 
+    def play_dtmf(self, session_id: str, connection_id: str, digits: str, options: dict = {}) -> requests.Response:
+        """
+        Plays a DTMF string into a session or to a specific connection
```
Comment (calthoff) [inline opentok/opentok.py:1465]: As someone new to this, I'm not clear what a DTMF string is or what it means to play a DTMF string.

File: opentok/opentok.py [hunk 4]
```diff
@@ -1447,3 +1460,41 @@ def mute(self, session_id: str, stream_id: str= "", options: dict = {}) -> reque
...
+        :param digits DTMF digits to play
+        Valid DTMF digits are 0-9, p, #, and * digits. 'p' represents a 500ms pause if a delay is
+        needed during the input process.
+
+        """
+
+        try:
+            if not connection_id:
+                url = self.endpoints.get_dtmf_all_url(session_id)
+                payload = {"digits": digits}
```
Comment (calthoff) [inline opentok/opentok.py:1482]: Why define payload twice if its the same in either case?

**Pull request merged**